.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
No need for this, personally.

Reply:
Reverted in [0b45908](https://github.com/Naval-Base/yuudachi/pull/1069/commits/0b4590852556bf1260e30d03fce2583cccf4029e). Sorry for the trouble!